bin/snare

```diff
@@ -34,6 +34,26 @@ from snare.utils import snare_helpers
 from snare.utils.snare_helpers import str_to_bool, print_color
 
 
+def check_meta_file(meta_info):
```

Comment:
Maybe it's better to put it in utils? https://github.com/mushorg/snare/blob/master/snare/utils/snare_helpers.py